README.md

```diff
@@ -28,20 +28,28 @@ Swagger is used to implement our OpenAPI specification and requires `mvn`, so be
 
 Beyond that, Node and Ruby are required to get started with documentation. The easiest way to work with the tooling is to install some version managers for some languages used in our toolchain.
 
+For this guide, we'll assume you're using `zsh`, if not you can replace the `~/.zshrc` with `~/.bash_profile`.
+
 For Node, we'll use [nvm](https://github.com/nvm-sh/nvm):
 
 ```shell
 brew install nvm
+# Brew doesn't set this up anymore, so we need to do this manually. See https://formulae.brew.sh/formula/nvm for up to date instructions if these fail
+mkdir -p ~/.nvm # You should create NVM's working directory if it doesn't exist:
+# Add the following to /Users/runner/.bash_profile or your desired shell configuration file:
+echo 'export NVM_DIR="$HOME/.nvm"' >> ~/.zshrc
+echo '[ -s "$(brew --prefix)/opt/nvm/nvm.sh" ] && \. "$(brew --prefix)/opt/nvm/nvm.sh"; >> ~/.zshrc # This loads nvm
```

Comment:
* Small bug: The above line has a typo — it is missing a closing single quote.
* In both the above line and the below line, the use of `\.` instead of `.` is odd to me — not sure why the backslash is there — but it works, so I guess that's okay.
* Same with the semicolon at the end — unneeded but harmless

Reply:
This was copy pasted from the NVM doc above (besides the missing single ')